src/core/zowe/core_for_zowe_sdk/request_handler.py

```diff
@@ -134,13 +134,12 @@ def __normalize_response(self):
 
         Returns
         -------
-        A bytes object if the response content type is application/octet-stream,
-        a normalized JSON for the request response otherwise
+        A bytes object at the format based on Content-Type header
```

Comment:
The object is not always bytes. We could say something like this:
```suggestion
        Response object at the format based on Content-Type header:
        - `bytes` if the response is binary data
        - object if the response is JSON text
        - `str` if the response is plain text
```